*** linux-4.14.5/scripts/package/mkspec	2017-12-10 04:40:45.000000000 -0800
--- linux-4.14.5-new/scripts/package/mkspec	2017-12-15 12:55:34.559169090 -0800
***************
*** 96,102 ****
  echo "cp vmlinux arch/powerpc/boot"
  echo "cp arch/powerpc/boot/"'$KBUILD_IMAGE $RPM_BUILD_ROOT'"/boot/vmlinuz-$KERNELRELEASE"
  echo "%else"
! echo 'cp $KBUILD_IMAGE $RPM_BUILD_ROOT'"/boot/vmlinuz-$KERNELRELEASE"
  echo "%endif"
  echo "%endif"
  
--- 96,105 ----
  echo "cp vmlinux arch/powerpc/boot"
  echo "cp arch/powerpc/boot/"'$KBUILD_IMAGE $RPM_BUILD_ROOT'"/boot/vmlinuz-$KERNELRELEASE"
  echo "%else"
! echo 'cp $KBUILD_IMAGE .' 
! echo 'iName=$(basename $KBUILD_IMAGE)'
! echo 'mv $iName '"vmlinuz-$KERNELRELEASE" 
! echo "mv vmlinuz-$KERNELRELEASE "'$RPM_BUILD_ROOT'"/boot"
  echo "%endif"
  echo "%endif"
  
